tests/test_api.rs

```diff
@@ -93,6 +93,63 @@ fn test_string() {
   drop(locker);
 }
 
+#[test]
+#[allow(clippy::float_cmp)]
+fn escapable_handle_scope() {
+  let g = setup();
+  let mut params = v8::Isolate::create_params();
+  params.set_array_buffer_allocator(v8::Allocator::new_default_allocator());
+  let mut isolate = v8::Isolate::new(params);
+  let mut locker = v8::Locker::new(&isolate);
+  isolate.enter();
+  v8::HandleScope::enter(&mut locker, |scope1| {
+    // After dropping EscapableHandleScope, we should be able to 
+    // read escaped values.  
+
+    let mut escapable_scope = v8::EscapableHandleScope::new(scope1);
+    let mut scope: v8::HandleScope = unsafe { std::mem::transmute_copy(&escapable_scope) };
+    let l1: Local<v8::Value> = cast(v8::Integer::new(&mut scope, -123));
+    let l1: Local<v8::Integer> = cast(escapable_scope.escape(l1));
```

Comment:
maybe don't write over the previous variable but use a different one

maybe put this all in a block to test scoping

Reply:
Done, now it looks super clean, I guess we could rewrite `HandleScope` to work the same way?